Spelling (#10976)
netdata/netdata: docs/guides/monitor/statsd.md

## Vendoring a new collector

> After all this hustle, not only did we illustrate how to visualize any data source in Netdata using the StatsD protocol, but we have also created a new collector in the process.

Author: This was actually harder than it looks, because I apparently mostly know this word phonetically and I initially picked the wrong correction for it...

Reviewer: This probably should be "hassle." That makes the most sense to me in context. @OdysLam?

Reviewer 2: @kickoke halp, what do you think is better?

Reviewer 3: > @kickoke halp, what do you think is better?

I think we should move away from all these words. "Hustle" seems like bragging, "hassle" seems like complaining. We should just go neutral on this and say "effort/work/decisions" or even just say "After all of this,"

Reviewer 4: 👍 +1 what @DShreve2 said :)

Reviewer 2: @kickoke or @DShreve2 can you reword it and make a suggestion so we can commit it using GitHub UI?

Author: ```suggestion
After all this, not only did we illustrate how to visualize any data source in Netdata using the StatsD protocol, but we have also created a new collector in the process.
```

## Vendoring a new collector

> After all this hustle, not only did we illustrate how to visualize any data source in Netdata using the StatsD protocol, but we have also created a new collector in the process.

Reviewer 4: ```suggestion
While we learned how to visualize any data source in Netdata using the StatsD protocol, we have also created a new collector.
```

> While using the same underlying collector-StatsD-every new `myapp.conf` file will in essence create a new data source and dashboard section for Netdata. While Netdata will load all the configuration files by default, it will **not** create dashboard sections or charts, unless it start receiving data for that particular data source. This means that we can now share our collector with the rest of the Netdata community. 

Reviewer 4: ```suggestion
As long as you use the same underlying collector, every new `myapp.conf` file will create a new data source and dashboard section for Netdata. Netdata loads all the configuration files by default, but it will **not** create dashboard sections or charts, unless it starts receiving data for that particular data source. This means that we can now share our collector with the rest of the Netdata community. 
```
